### ISSUE TITLE
aria-label value should be this.label perhaps?

### ISSUE THREAD

**Issue by snmishra** [json-editor/json-editor#1387]
I have a form that generates inputs as follows:
```html
<div class="form-control">
  <label class="required je-form-input-label" for="form[first_name]">First Name</label>
  <input type="text"
         pattern=".{1,}"
         name="form[first_name]"
         aria-label="form[first_name]" />
  <div></div>
</div>
```
[According to MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) `aria-label` should give the actual label. Which should, in this case, be "First Name" not the `name` attribute of the input.

The problematic line seems to be the linked one below.
https://github.com/json-editor/json-editor/blob/33f9f59a52a279e20faaef70b552e6a4b661502f/src/editors/string.js#L9C36-L9C36

**Comment by schmunk42** [comment]
Related:
- https://github.com/json-editor/json-editor/issues/997
- https://github.com/json-editor/json-editor/pull/1005

What does the reader say in the case above? Is it "form first name"?

If it only uses the label and you have a form with `father.first_name` and `child.first_name` it would be really helpful to have the "full path" and not only `first_name` - but my experience is very limited with screen readers.

> [According to MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) aria-label should give the actual label.

Where does the page say that? I only found this:
*If there is no visible name for the element you can reference, use aria-label to provide the user with a recognizable accessible name.*

**Comment by snmishra** [comment]
Thanks for looking into this. This is the bit from MDN doc on `arial-label` that tells me that it should be set to the label value when the label element is not visible.

_aria-label can be used in cases where text that could label the element is not visible. If there is visible text that labels an element, use [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) instead._

I have no experience about screen-readers. I only found out because Playwright uses the `aria-label` attribute in preference the `label` element value.

**Comment by schmunk42** [comment]
@itsrogil @cbonaudo You were in the discussion in the PR mentioned above, if you can help with this issue it would be cool.

**Comment by snmishra** [comment]
Fixed in 2.14.0
